lib/vector-vrl/web-playground/public/README.md

```diff
@@ -65,7 +67,42 @@ but they will either error (enrichment functions) or abort (all the others) at r
 - `get_enrichment_table_record`
 
 Functions from VRL stdlib that are currently not supported can be found
-with this [issue filter][vrl-wasm-unsupported-filter]
+with this [issue filter][vrl-wasm-unsupported-filter].
+
+### macOS Troubleshooting
```

Comment:
Weren't these added in https://github.com/vectordotdev/vector/pull/17824 too? Maybe this was accidentally included here?

Reply:
`wasm-pack build` essentially copies the `README.md` file into the generated package, in our case the `public` directory. Although we may not require `README.md` in the generated directory, the CLI args don't provide any way to overwrite this behavior.

Reply:
This just made me realize I also need to update https://github.com/vectordotdev/vector/blob/master/lib/vector-vrl/web-playground/netlify.toml.

Reply:
Aha, I see. Should we be checking in the generated README?

Reply:
No, it is unnecessary. And after my last commit we can also get rid of `public/.gitignore`.